bucket/locale-emulator.json

```
@@ -0,0 +1,44 @@
+{
+    "homepage": "https://pooi.moe/Locale-Emulator/",
+    "description": "Yet Another System Region and Language Simulator",
+    "version": "2.4.0.0",
+    "license": "LGPL-3.0,MS-PL",
+    "url": "https://github.com/xupefei/Locale-Emulator/releases/download/v2.4.0.0/Locale.Emulator.2.4.0.0.zip",
+    "hash": "94fc8351cf99941d1d8739e384753d8a085a075dda4a86bc6049ee7ec4f49720",
+    "bin": [
+        [
+            "LEGUI.exe",
+            "Locale-Emulator"
+        ],
+        "LEProc.exe"
+    ],
+    "shortcuts": [
+        [
+            "LEGUI.exe",
+            "Locale Emulator"
+        ],
+        [
+            "LEInstaller.exe",
+            "Locale Emulator Installer"
+        ],
+        [
+            "LEUpdater.exe",
+            "Locale Emulator Updater"
+        ]
+    ],
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\LEConfig.xml\")) {",
+        "    New-Item -Force -Path \"$persist_dir\\LEConfig.xml\" -ItemType file -Value $content | Out-Null",
```

Variable $content is not defined. Use $null or `''` instead

```suggestion
        "    New-Item -Force -Path \"$persist_dir\\LEConfig.xml\" -ItemType File -Value '' | Out-Null",
```